components/global/portal/io.cellery.observability.ui/src/components/overview/index.js

```diff
@@ -221,7 +221,8 @@ class Overview extends React.Component {
             legendOpen: false,
             listData: [],
             page: 0,
-            rowsPerPage: 5
+            rowsPerPage: 5,
+            cellIngress: []
```

Comment:
Change this to Ingresses (Since its plural). Please fix in other places as well accordingly.